cd/cd_dump.ixx

```diff
@@ -853,12 +880,35 @@ export bool redumper_dump_cd(Context &ctx, const Options &options, bool refine)
                         read_entry(fs_sub, (uint8_t *)sector_subcode_file.data(), CD_SUBCODE_SIZE, lba_index, 1, 0, 0);
                         ChannelQ Q_file;
                         subcode_extract_channel((uint8_t *)&Q_file, sector_subcode_file.data(), Subchannel::Q);
+
+
                         if(!Q_file.isValid())
                         {
                             write_entry(fs_sub, sector_subcode.data(), CD_SUBCODE_SIZE, lba_index, 1, 0);
                             if(inside_range(lba, error_ranges) == nullptr)
                                 --errors_q;
                         }
+
+                        if(options.check_subchannel_rw && lba >= 2 && lba + 2 < lba_overread)
+                        {
+                            read_entry(fs_sub, (uint8_t *)sector_subcode_rw_context.data(), CD_SUBCODE_SIZE, lba_index - 2, CD_RW_PACK_CONTEXT_SECTORS, 0, 0);
+
+                            if(!valid_rw_packs(sector_subcode_rw_context))
```

Comment:
There's a bug here, though I think it only affects the count: If Q was corrected above, and that also corrected R-W, then `sector_subcode` was already written to the file, so it will look here like it had already been correct. What's needed is to defer `write_entry` until after both the Q and R-W checks, probably set a `store_subcode` flag.

Reply:
It's going to be hard to keep the count accurate anyway, given that a sector may be marked correct by a read of the next sector. In the interest of not overcomplicating things I'm going to leave this as-is here, unless you'd prefer I handle it more thoroughly.

Reply:
I had an error in my initial `store_subcode` approach, the latest commit implements it correctly: the final number of R-W errors matches what will be initially counted on a subsequent run, before it would miscount if both Q and R-W were corrected on the same read.

It's still possible that a sector becomes valid only due to subsequent reads, but this should be rare (it would involve at least 3 errors in a pack, which I haven't yet seen), and anyway it will only affect the count. There's some outside chance that a subsequent read will make an earlier sector invalid so that it ought to be re-read, but this seems so unlikely that it isn't worth special handling. It's rather more likely that a re-read for Q makes R-W become invalid; this will be at least retried if there are retries left, but it will also throw off the count.